activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/resources/schema/connector-schema.json

```diff
@@ -52,9 +52,28 @@
         "outputs": { "$ref": "#/definitions/connector-parameters" }
       },
       "required": [ "id", "name" ]
+    },
+    "connector-event": {
```

Comment:
Instead of creating this new definition, you could rename the `connector-action` (line 36) definition to `connector-feature` as you have done in the converters and use that definition for actions (line 100) and events (line 103)